pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -180,6 +191,9 @@ func (r *ReconcileNodeNetworkConfigurationPolicy) Reconcile(request reconcile.Re
 
 	conditionsManager.NotifyMatching()
 
+	unlock := r.rollout.Lock()
+	defer unlock()
+
 	conditionsManager.NotifyProgressing()
```

Comment:
I think we have to mark enactments as progressing while they are lock.

Reply:
Do we want to mark as `Progressing` all nodes that are waiting to get the leadership or only the one that is the leader at the moment? I would vote for the latter for the nice effect...